packages/sdk/src/util/logger.ts

```diff
@@ -80,7 +80,7 @@ export function getLogLevel(): LogLevel {
 }
 
 class InternalLogger {
-  public level: LogLevel = LogLevel.Info;
+  public level: LogLevel = LogLevel.Error;
```

Comment:
This is a breaking change to developers. For the bug, seems we don't need to change default log level.
Can we check the default log level of other SDK (like MSAL) and then have some discussion around it before change it?

Reply:
Sure I agree. A related bug is https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/9921340

Reply:
Agree. We may need more discussion and investigation before making changes.